.github/scripts/junit.py

```diff
@@ -95,6 +96,9 @@ def parse_report(workspace_path, report_path, fp) -> Iterable[TestSuite]:
                 test_case_failed = False
             elif elem.tag == "failure":
                 failure_message = elem.get("message")
+                if failure_message:
+                    failure_message = failure_message.replace('\n', '<br>').replace('\r', '<br>')
+                    failure_message = html.escape(failure_message)
```

Comment:
I think these should be reversed. Otherwise the `<br>` will get escaped and not actually rendered as HTML

Reply:
Yeah, you are right, I should adjust the order